### ISSUE TITLE
Conditional Logic is not working Offline Envelope creation

### ISSUE THREAD

**Issue by pushp1010** [docusign/native-ios-sdk#142]
Hello @mmohareb @ashokds,

I am trying to use conditional logic  but Envlopemanage is not able to cache the template and giving me an error "Template has unsupported Recipient attributes"

```
templatesManager.cacheTemplate(
            withId: templateId,
            completion: { (error: Error?) in
                if let error = error {
                    completionHandler(.failure(error))
                } else {
                    completionHandler(.success(true))
                }
            })
```

Please help me with this

**Comment by mmohareb** [comment]
Hello @pushp1010 
Do you mean Conditional Tabs logic, where you show/hide tabs based on other tabs values (checked or not)?
Or do you mean Conditional Routing, where you can control the recipients flow based on tab values?

These 2 features are not supported in the SDK as of now, if you can work with your account executive to raise an enhancement request with the exact flow you need, this would help us prioritize this work to address your request.

**Comment by pushp1010** [comment]
> Hello @pushp1010 Do you mean Conditional Tabs logic, where you show/hide tabs based on other tabs values (checked or not)? Or do you mean Conditional Routing, where you can control the recipients flow based on tab values?
> 
Yes correct. What will be the rough timeline to built such feature

**Comment by mmohareb** [comment]
I think it will be better to request this through the account executive with an exact scenario to help prioritize this work
